nvtabular/framework_utils/tensorflow/tfrecords_to_parquet.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import gc
+
+import cudf
+import tensorflow as tf
+from cudf.core.column.lists import is_list_dtype
+from cudf.io.parquet import ParquetWriter
+from pandas_tfrecords.from_tfrecords import _get_feature_type, read_example
+from tqdm import tqdm
+
+
+def convert_tfrecords_to_parquet(
+    filenames, output_dir, compression_type="", chunks=100000, convert_lists=False
+):
+    """
+    Converts tfrecord files to parquet file format
+
+    Parameters
+    ----------
+    filenames: list
+        List of tfrecord filenames, which should be converted
+    output_dir: str
+        Output path where the parquet files will be stored
+    compression_type: str
+        Compression type of the tfrecords. Options: `""` (no compression), `"ZLIB"`, or `"GZIP"`
+    chunks: int
+        Chunks to convert tfrecords into parquet
+    convert_lists: Boolean
+        Output of tfrecords are lists. Set True to convert lists with fixed length to
+        individual columns in the output dataframe
+
+    """
+    # TODO: provide row_groupby_size parameter for parquet files
+    # TODO: optimize parquet files
```

Comment:
I think by using the cudf ParquetWriter we should have optimized parquet files - with sensible rowgroup sizes (assuming chunks is large enough), maybe we can remove this TODO?  
```suggestion
```

Reply:
Yes, I removed the ToDos